eng/tox/tox.ini

```diff
@@ -23,6 +23,12 @@ deps =
   {[tools]deps}
 
 
+[devtools]
+deps =
+  -e ../../../tools/azure-devtools
```

Comment:
My first instinct isn't to take a dev dep here. Is there a specific reason you're going after `-e` instead of the regular one?

Reply:
Fixed